src/plugins/poster/poster.js

```diff
@@ -42,12 +42,8 @@ export default class PosterPlugin extends UIContainerPlugin {
     this.listenTo(this.container, Events.CONTAINER_STOP, this.onStop)
     this.listenTo(this.container, Events.CONTAINER_PLAY, this.onPlay)
     this.listenTo(this.container, Events.CONTAINER_ENDED, this.onStop)
-    if (this.container.buffering) {
-      process.nextTick(() => this.onBuffering())
```

Comment:
Technically `buffering` doesn't need to be checked. But it means the play button disappears if the loading indicator is on, since they both check `buffering`, instead of play still being visible over the top if the user hasn't clicked it yet.